geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/ConvertCommandTest.scala

```diff
@@ -68,7 +68,12 @@ class ConvertCommandTest extends Specification with LazyLogging {
         command.params.config = conf
         command.params.spec = conf
         command.params.outputFormat = outFmt
-        command.params.file = File.createTempFile("convertTest", s".${outFmt.toString.toLowerCase}")
+        command.params.file = if (outFmt == DataFormats.Leaflet) {
+          File.createTempFile("convertTest", s".html")
```

Comment:
Lets add a line to clean this up...got a lot in my /tmp dir

Reply:
It's already cleaned up on ln 80-81

Reply:
our tests do seem to still leave tmp files around - we should track those down and fix em at some point